js/exporter/jspdf/v3/draw_utils.js

```diff
@@ -4,46 +4,58 @@ import { extend } from '../../../core/utils/extend';
 
 const defaultBorderLineWidth = 1;
 
-function drawCellsContent(doc, cellsArray) {
-    const docStyles = getDocumentStyles(doc);
+function drawCellsContent(doc, cellsArray, docStyles) {
     cellsArray.forEach(cell => {
-        // TODO: drawCellBackground(doc, cell);
+        drawCellBackground(doc, cell);
         drawCellText(doc, cell, docStyles);
     });
-    setDocumentStyles(doc, docStyles);
 }
 
-function drawCellText(doc, cell, docStyles) {
-    setCurrentFont(doc, cell, docStyles);
+function drawCellBackground(doc, cell) {
+    if(isDefined(cell.backgroundColor)) {
+        doc.setFillColor(cell.backgroundColor);
+        drawRect(doc, cell._rect.x, cell._rect.y, cell._rect.w, cell._rect.h, 'F');
+    }
+}
 
+function drawCellText(doc, cell, docStyles) {
     if(isDefined(cell.text) && cell.text !== '') { // TODO: use cell.text.trim() ?
+        const { textColor, font } = cell;
+        setTextStyles(doc, { textColor, font }, docStyles);
         drawTextInRect(doc, cell.text, cell._rect, cell.wordWrapEnabled, cell.jsPdfTextOptions);
     }
 }
 
-function drawCellsLines(doc, cellsArray) {
-    cellsArray.forEach(cell => {
-        // TODO: doc.setDrawColor(borderColor); // cell.borderColor OR docStyles.borderColor
-        drawBorders(doc, cell._rect, cell.drawLeftBorder, cell.drawRightBorder, cell.drawTopBorder, cell.drawBottomBorder);
-    });
+function drawCellsLines(doc, cellsArray, docStyles) {
+    cellsArray
```

Comment:
In both calling of the drawBorders uses the same parameters.
Arguments cycle is

> drawBorders(cell, docStyles) -> setLinesStyles(doc, { borderColor }, docStyles); -> setTextStyles ({ textColor, font }, docStyle)

Maybe it's better to compute resulting styles at drawCells and then pass it to the drawBorders method?

But it's discussable